openedx/core/lib/celery/routers.py

```diff
@@ -33,13 +33,6 @@ def route_for_task(self, task, args=None, kwargs=None):  # pylint: disable=unuse
         If None is returned from this method, default routing logic is used.
         """
         alternate_env = self.alternate_env_tasks.get(task, None)
-        if 'update_course_in_cache' in task:
-            log.info("TNL-5408: task={task}, args={args}, alternate_env={alt_env}, queues={queues}".format(
```

Comment:
Note to reviewers: This log statement is no longer needed since TNL-5408 was diagnosed and fixed.